src/components/Sum.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a, b }) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
+  </p>
```

Comment:
```suggestion
  <p> {`Sum of ${a} and ${b} is ${a + b}`} </p>
```
Мне кажется такая запись предпочтительнее, так как в будущем пропсов может стать больше и переносить каждый из них на новую строку еще и с такой вставкой пробела - это костыли.